media/js/firefox/whatsnew/whatsnew-61.js

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+This iteration of /whatsnew has multiple states:
+
+1. User is not logged in to Sync (or UITour JS fails/times out):
+    Display the FxA iframe
+
+2. User is logged in to Sync, but does not have any mobile devices set up:
+    Display App/Play store badges for Firefox Mobile
+
+    i. Send to Device widget is available in the user's locale:
+        Display the Send to Device widget, configured for Firefox Mobile
+    ii. Send to Device widget is *not* available in the user's locale:
+        Display a QR code for Firefox Mobile
+
+3. User is logged in to Sync, and *does* have a mobile device set up:
+
+    i. If they're in Germany, Austria, or Switzerland:
+        a. Send to Device widget is available in the user's locale:
+            Display the Send to Device widget, configured for Klar
+        b. Send to Device widget is *not* available in the user's locale:
+            Display a QR code for Klar
+    ii. If they're in any other country *or* geolocation fails
+        a. Send to Device widget is available in the user's locale:
+            Display the Send to Device widget, configured for Focus
+        b. Send to Device widget is *not* available in the user's locale:
+            Display a QR code for Focus
+
+*/
+
+(function(Mozilla, $) {
+    'use strict';
+
+    Mozilla.WNP61 = {
```

Comment:
whatsnew61 is largely a copy of whatsnew60, and I'm not sure yet what the plans are for whatsnew62 but I think it may repeat the same pattern again. We definitely should try to make this more reusable. It's a bit dirty right now.

Reply:
> Is this something that's likely to get reused?

This is the crux of the problem - we don't know. However, since this is going in-product soon-ish (I think?), maybe it's not a huge concern?

Reply:
Firstrun is going in-product, now /whatsnew (afaik)

Reply:
We've heard rumors whatsnew might soon go in-product as well but we should chase that rumor down and find out if it's real.